gbus/metrics/saga_metrics.go

```diff
@@ -29,3 +32,13 @@ func newSagaTimeoutCounter() prometheus.Counter {
 		Help:      "counting the number of timedout saga instances",
 	})
 }
+
+func newSagaLatencySummary() prometheus.SummaryVec {
```

Comment:
the `prometheus.NewSummaryVec` returns a pointer
```suggestion
func newSagaLatencySummary() *prometheus.SummaryVec {
```

Reply:
hmm, so why on the other metric (func newSagaTimeoutCounter() prometheus.Counter ) it's not a pointer?